web-side-nav-example-test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
This file needs to be in https://github.com/flutter/packages/tree/0bd0307e2187bfe873b6a3fe80219eb0799406c1/packages/go_router/example/test

and we use underscore for file name. same for the example file